.devcontainer/devcontainer.json

```
@@ -0,0 +1,14 @@
+{
+	"name": "dodona-tested",
```

```suggestion
  "name": "dodona-tested",
```
Use spaces instead of tabs